src/style.css

```diff
@@ -0,0 +1,100 @@
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
```

Comment:
Consider adding fallback fonts to `font-family` to ensure that the text is still readable if the custom font fails to load. 🆘